src/test/java/com/study/racing/step1/SetCollectionTest.java

```diff
@@ -0,0 +1,61 @@
+package com.study.racing.step1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.apache.logging.log4j.util.Strings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+
+    @Test
+    @DisplayName("요구사항 1")
+    public void test1() {
+        System.out.println(numbers.size());
```

Comment:
해당하는 함수는 void 가 출력되는지만 확인하는 테스트 코드 같아요~.
아래와같이 변경하면 좋을 것 같습니다.

```suggestion
       //given
        //when
        int size = numbers.size();
        //then
        assertThat(size).isEqualTo(3);
```